app/controllers/import_controller.rb

```diff
@@ -229,6 +230,16 @@ def import_and_validate_state_field(field, value, row_ind)
     raise ValidationError.new(err_msg, row_ind)
   end
 
+  def import_and_validate_language_field(field, value, row_ind)
+    return nil if value.blank?
+
+    val = normalize_and_get_language_name(value)
```

Comment:
Just want to make sure you're aware that this also allows 2-letter codes via import. I have no problems with that, but I didn't know if the PO team wanted to be more restrictive?

Reply:
Yep! That was intentional. The PO Team is okay with that